atmos.yaml

```diff
@@ -0,0 +1,69 @@
+# https://raw.githubusercontent.com/cloudposse/atmos/master/examples/complete/atmos.yaml
```

Comment:
It's a lot of opinionated config boilerplate, I agree. Let's think about if it really makes sense to have yet another automation tool here..